sdk/azcore/credential.go

```diff
@@ -6,14 +6,16 @@ package azcore
 import (
 	"context"
 	"encoding/json"
+	"net/url"
 	"time"
 )
 
 // AuthenticationPolicyOptions contains various options used to create a credential policy.
 type AuthenticationPolicyOptions struct {
 	// Scopes is the list of OAuth2 authentication scopes used when requesting a token.
 	// This field is ignored for other forms of authentication (e.g. shared key).
-	Scopes []string
+	Scopes        []string
+	AuthorityHost url.URL
```

Comment:
Check if this is actually needed

Reply:
I don't think so, since we pass in a custom authority host to the credential through Identity Client Options. If someone has a different perspective on this I'm glad to hear, but I'll remove the field for now and resolve this.